cluster/components/kuberay/README.md

```
@@ -0,0 +1,94 @@
+# Step 2: Send a request to the fruit stand app.
+curl -X POST -H 'Content-Type: application/json' rayservice-sample-serve-svc:8000/fruit/ -d '["MANGO", 2]'
+# [Expected output]: 6
+
+# Step 3: Send a request to the calculator app.
+curl -X POST -H 'Content-Type: application/json' rayservice-sample-serve-svc:8000/calc/ -d '["MUL", 3]'
+# [Expected output]: "15 pizzas please!"
+```
+
+**Clean up the Kubernetes cluster**
+
+```
```

Markdownlint

Fenced code blocks should have a language specified

(MD040, fenced-code-language)